src/components/property-detail/hero/RequestInspectionPopupScreen.js

```diff
@@ -0,0 +1,252 @@
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 17px;
+`;
+
+const Checkbox = styled.input`
+  margin-left: 0;
+`;
+
+const CheckboxContent = styled.div`
+  margin-left: 8px;
+`;
+
+const Divider = styled.div`
+  margin: 0 auto;
+  width: 370px;
+  height: 1px;
+  background-color: #D2D5DA;
+  margin: 20px 0;
+`;
+
+const PersonalDetailsInputContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const PersonalDetailsInput = styled.input`
+  width: 38%;
+  border-radius: 4px;
+  border-color: rgb(195, 200, 206);
+  border-style: solid;
+  border-width: 1px;
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+  margin: 10px 0 5px 0;
+`;
+
+const DropdownListContainer = styled.input`
+  box-sizing: border-box;
+  width:100%;
+  margin: 10px 0 5px 0;
+  border-radius: 4px;
+  border-color: rgb(195, 200, 206);
+  border-style: solid;
+  border-width: 2px;
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+`;
+
+const CommentContainer = styled.input`
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 4px;
+  border-color: rgb(195, 200, 206);
+  border-style: solid;
+  border-width: 1px;
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+  margin: 10px 0 5px 0;
+`;
+
+const SubmitButtonContainer = styled.button`
+  width: 200px;
+  padding: 12px;
+  margin-top: 15px;
+  background-color: rgb(228, 0, 43);
+  color: white;
+  text-align: center; 
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 13px;
+  border-radius: 4px;
+  border: none;
+  &:hover {
+    cursor: pointer;
+    background-color: rgb(190, 0, 18);
+  }
+`;
+
+const SubmitRemarksContainer = styled.div`
+  text-align: center;
+  margin-top: 15px;
+`;
+
+const RequestInspectionPopupScreen = (props) => {
+  return (
+    <div>
+      <Container>
+        <div>
+          <CloseButton onClick={props.closeRequestAnInspectinPopUp}>X CLOSE</CloseButton>
+          <SubContainer>
+            <BrandingBar listingCompany={props.listingCompany} height="40" />
+            <ContentContainer>
+              <Title>Book an Inspection</Title>
+              <Address>{props.address}</Address>
+              <SubTitle>Inspection day and time</SubTitle>
+              <DayAndTimeContainer>
+                <Date type="date" placeholder="Day" /><Time type="time" placeholder="Time" />
+              </DayAndTimeContainer>
+              <CheckboxContainer>
+                <Checkbox type="checkbox" />
+                <CheckboxContent>
+                  <SubTitle>I can't find a suitable day or time</SubTitle>
+                  <RemarksTitle>One of our agents will get in touch to arrange another time</RemarksTitle>
```

Comment:
We can change it to `label` as well like 
```js
const TimeNotSuitableRemarksLabel = styled.label`
  display: block;
  font-family: "PangeaRegular";
  font-size: 11px;
  font-weight: 300;
  color: rgb(97, 97, 97);
`;
```

Reply:
Then add `for="time-no-suitable-label"` to make the checkbox selectable.

Reply:
Wrap SubTitle & RemarksTitle by <label for="time-not-suitable-label"></label>